mongodb/src/test/java/org/apache/calcite/adapter/mongodb/MongoAdapterTest.java

```diff
@@ -217,6 +217,17 @@ private CalciteAssert.AssertThat assertModel(URL url) {
                 "{$project: {STATE: '$state', ID: '$_id'}}"));
   }
 
+  @Test void testJoin() {
+    assertModel(MODEL)
+        .query("select b.state, a.id from zips as a join zips as b on a.id=b.id where a.id='02401' "
```

Comment:
We should add another test case so that we can make a comparison. For instance:
`select b.state, a.id from zips as a join zips as b on a.id=b.id`

Reply:
Done，add the case. @NobiGo